app/ducks/walletActions.js

```diff
@@ -283,7 +282,8 @@ async function parseInputsOutputs(net, tx) {
     // affect the spendable balance of the wallet.
     // TODO: Transfer, Finalize, etc will eventually need to go here
     if (covenant.action === 'RENEW'
-        || covenant.action === 'UPDATE') {
+      || covenant.action === 'UPDATE' ||
+      covenant.action === 'TRANSFER') {
```

Comment:
FINALIZE here, spendable balance delta is zero.
Can remove comment on line 283!